ang/civicase/case/actions/services/edit-tags-case-action.service.js

```diff
@@ -82,13 +82,27 @@
     function getTags () {
       return civicaseCrmApi('Tag', 'get', {
         sequential: 1,
-        used_for: { LIKE: '%civicrm_case%' },
-        options: { limit: 0 }
+        used_for: 'Cases',
+        options: { limit: 0, sort: 'name ASC' }
       }).then(function (data) {
-        return data.values;
+        return filterTags(data.values);
       });
     }
 
+    /**
+     * Removes tags that are not selectable from the array.
+     *
+     * @param {Array} tags tags
+     * @returns {Array} filtered tags array.
+     */
+    function filterTags (tags) {
+      _.remove(tags, function (tag) {
+        return tag.is_tagset === '0' && tag.is_selectable === '0';
```

Comment:
There is a `isTruthy` service, which should be used here. 
1. Inject the service here https://github.com/compucorp/uk.co.compucorp.civicase/blob/4d1e478281602ca724a075edb193542b148e3c88/ang/civicase/case/actions/services/edit-tags-case-action.service.js#L12.
2. Something like `return !isTruthy(tag.is_tagset) && !isTruthy(tag.is_selectable)`